.github/workflows/molecule-install-xnat.yml

```
@@ -4,7 +4,7 @@ on:
     paths:
       - "roles/xnat/**"
       - "roles/xnat_container_service/**"
-      - ".github/workflows/molecule.yml"
+      - "**/molecule.yml"
```

```suggestion
     - "tests/molecule/*install_xnat/**/*"
```
The playbooks don't use the base configs - they have their own molecule setups, so I _think_ this should make the workflow run when any of the relevant configuration or variable files are change. And the `molecule.yml` reusable workflow isn't used for running by this workflow (I think because some additional packages are needed)